pt/endpoints/powerdata/model.py

```diff
@@ -9,133 +11,113 @@
 
-    def __init__(self, data_struct):
-        self.uuid = data_struct["uuid"]
-        self.field = data_struct["field"]
-        self.updated_at = data_struct["updated_at"]
-        self.history_id = data_struct["history_id"]
-        self.address = data_struct["address"]
+    # def __init__(self, data_struct):
+    #     self.uuid = data_struct["uuid"]
+    #     self.field = data_struct["field"]
+    #     self.updated_at = data_struct["updated_at"]
+    #     self.history_id = data_struct["history_id"]
+    #     self.address = data_struct["address"]
 
 
 class Demand(PowerData):
     __tablename__ = "demand"
-    uuid = db.Column(db.String(40), db.ForeignKey("powerdata.uuid"), primary_key=True)
+    uuid = db.Column(UUID(), db.ForeignKey("powerdata.uuid"), primary_key=True)
     grid = db.Column(db.Float)
     pv = db.Column(db.Float)
     building = db.Column(db.Float)
     ess = db.Column(db.Float)
     ev = db.Column(db.Float)
     __mapper_args__ = {"polymorphic_identity": "Demand"}
 
-    # fmt: off
-    def __init__(self, data_struct, history_id, address):
-        mother = {
-            'uuid': data_struct['id'],
-            'field': data_struct['field'],
-            'updated_at': data_struct['updated_at'],
-            'history_id': history_id,
-            'address': address,
-        }
-        super(Demand, self).__init__(mother)
-        self.grid = data_struct['grid']
-        self.building = data_struct['building']
-        self.ess = data_struct['ess']
-        # pylint: disable=C0103
-        self.pv = data_struct['pv']
-        self.ev = data_struct['ev']
-        # pylint: enable=C0103
-    # fmt: on
-
 
 class ESS(PowerData):
     __tablename__ = "ess"
-    uuid = db.Column(db.String(40), db.ForeignKey("powerdata.uuid"), primary_key=True)
+    uuid = db.Column(UUID(), db.ForeignKey("powerdata.uuid"), primary_key=True)
     cluster = db.Column(db.Integer)
     power_display = db.Column(db.Float)
     __mapper_args__ = {"polymorphic_identity": "ESS"}
 
-    def __init__(self, data_struct, history_id, address):
-        mother = {
-            "uuid": data_struct["id"],
-            "field": data_struct["field"],
-            "updated_at": data_struct["updated_at"],
-            "history_id": history_id,
-            "address": address,
-        }
-        super(ESS, self).__init__(mother)
-        self.cluster = data_struct["cluster"]
-        self.power_display = data_struct["power_display"]
+    # def __init__(self, data_struct, history_id, address):
+    #     mother = {
+    #         "uuid": data_struct["id"],
+    #         "field": data_struct["field"],
+    #         "updated_at": data_struct["updated_at"],
+    #         "history_id": history_id,
+    #         "address": address,
+    #     }
+    #     super(ESS, self).__init__(mother)
+    #     self.cluster = data_struct["cluster"]
+    #     self.power_display = data_struct["power_display"]
```

Comment:
Again, we can remove this section instead of commenting it.

Reply:
This is not the latest version, all comments have been remove :)